spec/dummy/app/avo/resources/project_resource.rb

```diff
@@ -17,7 +17,7 @@ class ProjectResource < Avo::BaseResource
   field :users_required, as: :number, min: 10, max: 1000000, step: 1, index_text_align: :right
   field :started_at, as: :date_time, name: "Started", time_24hr: true, relative: true, timezone: "EET", nullable: true
   field :description, as: :markdown, height: "350px"
-  field :files, as: :files, translation_key: "avo.field_translations.file", is_image: true, direct_upload: true
+  field :files, as: :files, translation_key: "avo.field_translations.file", is_image: true, direct_upload: true, accept: "image/*"
```

Comment:
Also, could you remove this, please? This field could be anything really.

I appreciate you added the `accept` attribute to the `post_resource`. Thank you!